charts/gardener/gardenlet/charts/runtime/templates/rbac.yaml

```
@@ -15,17 +68,327 @@ metadata:
     heritage: "{{ .Release.Service }}"
 rules:
 - apiGroups:
-    - "*"
+  - ""
+  resources:
+  - pods
+  verbs:
+  - list
+  - watch
+  - delete
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  - namespaces
+  - secrets
+  - serviceaccounts
+  - services
+  verbs:
+  - create
+  - delete
+  - deletecollection
+  - get
+  - list
+  - watch
+  - patch
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - persistentvolumeclaims
+  verbs:
+  - get
+  - list
+  - watch
+  - patch
+  - update
+- apiGroups:
+  - ""
+  resources:
+  - persistentvolumeclaims
+  resourceNames:
+  - alertmanager-db-alertmanager-0
+  - loki-loki-0
+  - prometheus-db-prometheus-0
+  verbs:
+  - delete
+- apiGroups:
+  - admissionregistration.k8s.io
+  resources:
+  - mutatingwebhookconfigurations
+  - validatingwebhookconfigurations
+  verbs:
+  - create
+- apiGroups:
+  - admissionregistration.k8s.io
+  resources:
+  - mutatingwebhookconfigurations
+  resourceNames:
+  - vpa-webhook-config-seed
+  verbs:
+  - get
+  - update
```

When the Seed is with

```yaml
spec:
  settings:
    verticalPodAutoscaler:
      enabled: false
```

then gardenlet will try to delete this webhook. 

https://github.com/gardener/gardener/blob/9df08ae3ae865f96db4161eb4ad01ced73107f3e/pkg/operation/common/utils.go#L280.

```
time="2021-03-26T13:56:05+02:00" level=error msg="seed is not yet ready: condition \"Bootstrapped\" has invalid status False (expected True) due to BootstrappingFailed: mutatingwebhookconfigurations.admissionregistration.k8s.io \"vpa-webhook-config-seed\" is forbidden: User \"system:serviceaccount:garden:gardenlet\" cannot delete resource \"mutatingwebhookconfigurations\" in API group \"admissionregistration.k8s.io\" at the cluster scope" operation=reconcile shoot=garden-foo/aws-local
```

```suggestion
  - vpa-webhook-config-seed
  verbs:
  - get
  - update
  - delete
```